content/influxdb/v2.0/upgrade/v1-to-v2/automatic-upgrade.md

```diff
@@ -172,6 +172,10 @@ InfluxDB 2.0 doesn't support [in-memory (`inmem`) indexing](/influxdb/v1.8/admin
 - `max-series-per-database`
 - `max-values-per-tag`
 
+### Interactive shell
+
+There is no interactive shell you can build on in InfluxDB 2.0. To build a command line Flux REPL, [follow these instructions](https://github.com/influxdata/flux/blob/master/README.md#getting-started) to compile and build from source. 
+
```

Comment:
```suggestion
```suggestion
The InfluxDB 2.x `influx` CLI does not include an interactive shell for executing InfluxQL queries or Flux queries. For information about executing InfluxQL queries, see [Query data with InfluxQL](/influxdb/v2.0/query-data/influxql/). To build a command line Flux REPL, [follow these instructions](https://github.com/influxdata/flux/blob/master/README.md#getting-started) to compile and build from source. 
```

```

Reply:
Maybe for last sentence above:
To execute Flux queries in an interactive shell, [compile and build a command line Flux REPL from source](https://github.com/influxdata/flux/blob/master/README.md#getting-started).